media-sound/jack/files/jack-3.1.1-python26.patch

```diff
@@ -0,0 +1,12 @@
+--- a/jack_functions.py
```

Comment:
Ok, that explains it. You can either modify the old file, i.e. change the header there, or just remove it along with the old ebuild.